source/fe/fe_nedelec.cc

```diff
@@ -3201,7 +3201,7 @@ FE_Nedelec<dim>::convert_generalized_support_point_values_to_dof_values(
           // are obtained by solving
           // a linear system of
           // equations.
-          if (this->degree - 1 > 1)
+          if (this->degree > 1)
```

Comment:
Does this fix imply that the comment above is also wrong?

Reply:
The comment above, though confusing, is technically correct. The comment refers to the degree of the FE_Nedelec cell, whereas this->degree returns the highest polynomial degree of the cell (which is FE_Nedelec_degree + 1).

Reply:
Ah, interesting. Would you mind clarifying the comment in this regard by pointing out the difference between the degree of the element and that `this->degree` is the *polynomial* degree?

I suspect that it is exactly this difference that led to the bug in the first place.